libvast/include/vast/system/transformer.hpp

```diff
@@ -15,6 +15,7 @@
 #include "vast/system/sink.hpp"
 #include "vast/table_slice.hpp"
 
+#include <caf/broadcast_downstream_manager.hpp>
```

Comment:
This include is needed. The file uses broadcast_downstream_manager but in CAF 0.18.6 lack of it leads to compilation error.
It is included by some other caf dependency here i believe

Reply:
Since this seems unrelated to the changes of the rest of the commit: It would be great if you could put such fixes in a separate commit next time.